BingMaps/rest-services/status-codes-and-error-handling.md

```diff
@@ -17,7 +18,7 @@ ms.service: "bing-maps"
 
 Information on status codes and error handling.
 
-## Handling empty responses
+## Handling Empty Responses
```

Comment:
The capitalization in headers is wrong all over the place in the Bing Maps Docs See [here](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28718) "Microsoft style uses sentence-style capitalization. That means everything is lowercase except the first word of a sentence "

```suggestion
## Handling empty responses
```